net/tcp/tcp_conn.c

```diff
@@ -656,11 +656,13 @@ FAR struct tcp_conn_s *tcp_alloc(uint8_t domain)
            * in the socket layer.
```

Comment:
since `crefs = 0;` is now moved after `SO_LINGER` handling, should we consider this change as one that address "REVISIT" comment here?

Reply:
Yes, REVISIT comment could be removed directly